MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include *.txt
 include LICENSE
 include README.md
+recursive-include nlp_primitives/data/nltk-data/ *
```

Comment:
This will include the data files once they've been un-tarred by the pre-sdist untarring command in `setup.py`